### ISSUE TITLE
[Suggestion] Hook Walls module countdown into the scoreboard module to display time until the wall drops.

### ISSUE THREAD

**Issue by mibby** [slipcor/pvparena#148]
Alternatively allow it to be displayed via bossbar countdown or action bar title. Though long timers may be better on the scoreboard instead.

**Comment by slipcor** [comment]
@mibby alright, the base work is actually done for some time. I asked you that question somewhere but I forgot where.

How should it look like? Remember you only have like 14 characters... what should the message be? :)

**Comment by mibby** [comment]
@slipcor Are you sure there is a character length of 14? FeatherBoard seems to display much more, with lines that have numerous color codes utilized.

![link](https://proxy.spigotmc.org/3655e00ae01c8752c2f3c3ad61c63f1120634fc1?url=http%3A%2F%2Fwww.mvdw-software.com%2Fwp-content%2Fuploads%2F2015%2F10%2FFeatherBoard31.gif)

I'm still waiting on when you properly support scoreboard customization so you could display total players, teams, team score, individual kills, etc. in any order you'd like. Though for time, I'd probably suggest something like 

&cWalls xx:xx

or 

&cWalls drop in: &axx:xx

xx:xx being minutes:seconds

**Comment by slipcor** [comment]
well okay... so you want configurable text? Please give me one example, fully fleshed out for any arena gamemode, and that will be the default, and people have to switch it up a bit depending on the game mode

I will then just blindly assume that it works for 32 characters..... if it breaks, bad luck :P

**Comment by mibby** [comment]
This ticket might actually be better to wait for full scoreboard customization with module variable hooks. So mainly just serving as a ticket reminder to add a walls variable for time remaining until the walls drop. So it has the capability of being displayed.

We'll discuss scoreboard customization further on IRC and this ticket https://github.com/slipcor/pvparena/issues/11

**Comment by slipcor** [comment]
Done in v1.3.3.225 - I reintroduced custom scoreboard entries, and implemented them in the walls module, configurable via language :)

I JUST saw that you went with MM:SS but the way you suggested does absolutely not work. The entry character limit is 14 characters, I just had an error proving me that the other day. If some other plugin does this I bet it is protocol or packet hacking :P

**Comment by mibby** [comment]
Feature implemented and working as intended according to last test. Will open a new ticket for any newly discovered issues.
